### ISSUE TITLE
Pending?

### ISSUE THREAD

**Issue by pesmi30** [elupus/hass_nibe#203]
I have the addon running fine. Thank you for it.
But the status of the heatpump is not very up-to-the-moment.
(When I look at the Nibe uplink-app on my phone, the status seems always up-to-the-moment....)

The 'connection status' often is 'PENDING' (most of the time for some 7 minutes) and then it's 'ONLINE' for a minute or so (and the sensors are updated)

Is it a configuration parameter I have missed?

**Comment by elupus** [comment]
Not really. It will update as fast as it can. But there is a rate limit on the nibe API limiting how often we are allowed to request data 

The more sensors active/available the slower the rate for each individual sensor.

Its not likely up to the moment on the phone either. The pump just reports periodically too.

**Comment by pesmi30** [comment]
Ok, thanks…
I’ll consoider the ‘connection status’ in automations (when necessary).

And when I have gathered enough courage, maybe I’ll build https://gathering.tweakers.net/forum/list_messages/2106834  (sorry its dutch)

 

Van: Joakim Plate ***@***.***> 
Verzonden: zondag 28 januari 2024 10:03
Aan: elupus/hass_nibe ***@***.***>
CC: pesmi30 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [elupus/hass_nibe] Pending? (Issue #203)

 

Not really. It will update as fast as it can. But there is a rate limit on the nibe API limiting how often we are allowed to request data

The more sensors active/available the slower the rate for each individual sensor.

Its not likely up to the moment on the phone either. The pump just reports periodically too.

**Comment by elupus** [comment]
Or you can buy a lilygo t can (https://github.com/Xinyuan-LilyGO/T-CAN485) of the shelf an run the builtin nibe component in HA.

**Comment by elupus** [comment]
Ps. You wont get the nibepi automatic price adaptive stuff with that. You have to automate it yourself.
